### PR TITLE
Cp 28368

### PR DESCRIPTION
**Pull request by lippirk**

### CODE REVIEW

**Comment by lippirk** [comment]
corresponds to https://github.com/xapi-project/xapi-test-utils/pull/6

File: ocaml/tests/test_common.ml [hunk 1]
```diff
@@ -15,27 +15,25 @@
...
-    assert_failure (Printf.sprintf "Function didn't raise expected API error %s" code)
+    Alcotest.fail (Printf.sprintf "Function didn't raise expected API error %s" code)
   with Api_errors.Server_error (c, a) ->
-    assert_equal ~printer:id ~msg:"Function raised unexpected API error" code c;
+    Alcotest.check Alcotest.string "Function raised unexpected API error" code c;
     match args with
     | None -> ()
     | Some args ->
-      assert_equal ~printer:Test_printers.(list string) ~msg:"Function raised API error with unexpected args" args a
+      Alcotest.check (Alcotest.list Alcotest.string) "Function raised API error with unexpected args" args a
```
Comment (edwintorok) [inline ocaml/tests/test_common.ml]: you could write `Alcotest.(list string)`

Comment (robhoes) [reply ocaml/tests/test_common.ml]: Or `Alcotest.(check (list string) ...)`

**Comment by robhoes** [comment]
Looks good! Now we just need to update xs-opam before we merge this.

**Pull request merged**